pegasus/sites.v3/hourofcode.com/views/how-to/steps/steps_events.haml

```diff
@@ -1 +1,119 @@
+:ruby
+  steps_educators_index = 7
 
+  steps_educators_heading = [
+    hoc_s(:hoc_how_to_step_label_register_event),
+    hoc_s(:hoc_how_to_step_label_choose_activities),
+    hoc_s(:hoc_how_to_step_label_prepare_your_event),
+    hoc_s(:hoc_how_to_step_label_plan_your_hour),
+    hoc_s(:hoc_how_to_step_label_get_coding),
+    hoc_s(:hoc_how_to_step_label_after_event),
+    hoc_s(:hoc_how_to_step_label_beyond_hoc)
+  ]
+
+  steps_educators_desc = [
+    hoc_s(:hoc_how_to_step_events_register_desc_01, markdown: :inline, locals:{share_url: "https://docs.google.com/forms/d/e/1FAIpQLSdD90kD2nkNZqTUMcAb2cQdOhUv99Q5XDQmkLDec25yZJHYhw/viewform"}),
+    hoc_s(:hoc_how_to_step_choose_activities_desc_01),
+    hoc_s(:hoc_how_to_step_events_prepare_desc_01),
+    hoc_s(:hoc_how_to_step_plan_hour_desc_01),
+    hoc_s(:hoc_how_to_step_events_get_coding_desc_01),
+    hoc_s(:hoc_how_to_step_events_after_desc_01),
+    hoc_s(:hoc_how_to_step_beyond_hoc_subhead)
+  ]
+
+%section.steps
+  .wrapper
+
+    = view :"how-to/steps/side-nav/side_nav_events"
+
+    %div.steps-wrapper
+      - steps_educators_index.times do |index|
```

Comment:
If you wanted to get really fancy, I think you could do something like

```
steps_educators_heading.zip(steps_educators_desc).each_with_index do |heading, desc, index|
 # Your logic here
```

Might be overkill as I doubt these pages will get many updates outside of HoC season.

Reply:
Oh whoa this does seem fancy — I'll keep it as-is for now since we're in a time crunch. I'd need time to figure out how this works, but love the suggestion!